mainPlugin/src/main/java/fr/il_totore/manadrop/Description.java

```diff
@@ -56,8 +56,8 @@ public void write(ConfigurationSection section) {
         description.ifPresent(value -> section.set("description", value));
         website.ifPresent(value -> section.set("website", value));
         depend.ifAllPresent(0, values -> section.set("depend", values));
-        softDepend.ifAllPresent(0, values -> section.set("softDepend", values));
-        loadBefore.ifAllPresent(0, values -> section.set("loadBefore", values));
+        softDepend.ifAllPresent(0, values -> section.set("softdepend", values));
```

Comment:
Effectively, I misspelled the attributes. Sorry for not seeing your issue and thank you for the PR